openreview/conference/matching.py

```diff
@@ -115,7 +115,7 @@ def _create_edge_invitation(self, edge_id):
         edge_head_query = {
             'invitation' : self.conference.get_blind_submission_id()
         }
-        if 'Custom_Load' in edge_id:
```

Comment:
these commits belong to another PR

Reply:
Correct, I am expecting the PR https://github.com/openreview/openreview-py/pull/645 to be merged first. Without these changes the branch would fail the tests.